test/integration/quota/quota_test.go

```diff
@@ -484,10 +486,10 @@ func TestQuotaLimitService(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unexpected err: %v", err)
 	}
-	go resourceQuotaController.Run(ctx, 2)
+	go resourceQuotaController.Run(ctx, 1)
 
 	// Periodically the quota controller to detect new resource types
-	go resourceQuotaController.Sync(discoveryFunc, 30*time.Second, ctx.Done())
+	go resourceQuotaController.Sync(discoveryFunc, controllerSyncPeriod, ctx.Done())
```

Comment:
if we're not adding/removing defined types (e.g. CRDs) in this test, this sync should have no effect, right? it's not clear to me why we're shortening this from happening every 30 seconds to happen every 5 seconds

Reply:
sync period has been reverted to 30s